src/ocl/ocl.client.ts

```diff
@@ -1,30 +1,31 @@
-import { Injectable, Logger } from '@nestjs/common';
+import { Inject, Injectable } from '@nestjs/common';
 import * as axios from 'axios';
-import { OclConcept } from '../../common/interfaces/ocl-concept.interface';
-import { apiConfig } from './config';
-import { ListOfOclConcepts } from '../../common/interfaces/list-of-ocl-concepts.interface';
-import { ProductNotFoundException } from '../../common/exceptions/product-code-does-not-exist.exception';
-import { OclClientException } from '../../common/exceptions/ocl-client.exception';
-import { ProductNotFoundInSystemException } from '../../common/exceptions/product-does-not-exist-in-the-specified-system.exception';
 import { AxiosError } from 'axios';
+import { WINSTON_MODULE_PROVIDER } from 'nest-winston';
+import { Logger } from 'winston';
+import { OclConcept } from '../common/interfaces/ocl-concept.interface';
+import { apiConfig } from './ocl.config';
+import { ListOfOclConcepts } from '../common/interfaces/list-of-ocl-concepts.interface';
+import { ProductNotFoundException } from '../common/exceptions/product-code-does-not-exist.exception';
+import { OclClientException } from '../common/exceptions/ocl-client.exception';
+import { ProductNotFoundInSystemException } from '../common/exceptions/product-does-not-exist-in-the-specified-system.exception';
```

Comment:
if the file name inherits the name of the Exception itself, it would be just fine and shorter too.

Reply:
That is true but it does not feel right.
```javascript
throw new ProductNotFound();
throw new ProductNotFoundException();
// the latter feels right for me
```